spyder/plugins/ipythonconsole.py

```diff
@@ -621,6 +621,14 @@ def setup_page(self):
         prompts_layout.addLayout(prompts_g_layout)
         prompts_group.setLayout(prompts_layout)
 
+        # Windows adjustments
+        windows_group = QGroupBox(_("Windows adjustments"))
+        show_subprocess_window = newcb(_("Show subprocess windows"),

```

Comment:
I think it'b be better to phrase this like

> Show command line windows generated by the `subprocess` module.

@CAM-Gerlach, do you have another suggestion?

Reply:
@ccordoba12 I agree with your suggestion, with the slight tweak to "command line *output* windows" to make it clear what these windows would be showing (that would be of interest to the user).

Also, I would consider inverting the name to "Hide" rather than "Show" (and therefore making it checked by default, so the default still hides the windows), since this is what Spyder is actually doing relative to a standard Python interpreter particularly from the user's perspective.

Reply:
I agree, thanks @CAM-Gerlach!